### ISSUE TITLE
If `nbqa black` fails, then `nbqa black --nbqa-diff` doesn't show anything

### ISSUE THREAD

**Issue by MarcoGorelli** [nbQA-dev/nbQA#483]
```
$ python -m nbqa black 100days-Day-063.ipynb --nbqa-diff
$ python -m nbqa black 100days-Day-063.ipynb --nbqa-mutate
error: cannot format 100days-Day-063.ipynb: Cannot parse: cell_7:1:5: nbqa black 100days-Day-062.ipynb --nbqa-diff
Oh no! 💥 💔 💥
1 file failed to reformat.
```

**Comment by MarcoGorelli** [comment]
this is fine, --nbqa-diff replaces the output anyway

**Comment by StefanBrand** [comment]
I don't think it's fine. I was just wondering whether my code is so perfect that there would be no diff at all. :stuck_out_tongue:

`--nbqa-diff` should show an error if formatting does not work.

**Comment by MarcoGorelli** [comment]
Hi @StefanBrand - thinking about this again, you're right, showing nothing is misleading. Pull requests to fix this are welcome! Else I hope to get time to work on it in a few weeks' time

**Comment by MarcoGorelli** [comment]
@StefanBrand fixed in version 0.6.0
